mdns.go

```diff
@@ -165,8 +168,16 @@ func (m *MDNS) BrowseMDNS() {
 		}
 	}(srvEntriesCh)
 
-	queryService("_workstation._tcp", entriesCh)
-	queryService("_etcd-server-ssl._tcp", srvEntriesCh)
+	var iface net.Interface
+	if m.bindAddress != "" {
+		foundIface, err := publisher.FindIface(net.ParseIP(m.bindAddress))
+		if err != nil {
+			log.Errorf("Failed to find interface for '%s'\n", m.bindAddress)
```

Comment:
Should this now return an error? I see we log but should we only assign `iface = foundIface` when this is nil?

Reply:
Yep, you're right. I'll fix that.